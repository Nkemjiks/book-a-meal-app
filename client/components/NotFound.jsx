import React from 'react';
import '../scss/notFound.scss';

/**
 * 404 page
 *
 * @returns {JSX} JSX representation of component
 */
const NotFound = () => (
  <div>
    <div className="center landing">
      <h1 id="h1">404</h1>
      <h2 id="h2">Page Not Found</h2>
    </div>
  </div>
);

export default NotFound;
