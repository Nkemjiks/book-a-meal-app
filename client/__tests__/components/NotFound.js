import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import NotFound from '../../components/NotFound';

Enzyme.configure({ adapter: new Adapter() });

describe('LandingPage Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders page Landing Page', () => {
    const wrapper = shallow(<NotFound />);
    expect((wrapper).find('div').length).toBe(1);
    expect((wrapper).find('h1').first().text()).toBe('404');
    expect((wrapper).find('h2').first().text()).toBe('Page Not Found');
  });
});
