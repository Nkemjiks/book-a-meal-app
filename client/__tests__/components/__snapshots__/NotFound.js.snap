// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Component renders correctly 1`] = `
<div
  className="center landing"
>
  <h1
    id="h1"
  >
    404
  </h1>
  <h2
    id="h2"
  >
    Page Not Found
  </h2>
</div>
`;
